packages/twenty-server/test/integration/billing/suites/billing-controller.integration-spec.ts

```diff
@@ -0,0 +1,280 @@
+      billing_cycle_anchor_config: null,
+      billing_thresholds: null,
+      cancel_at: null,
+      cancellation_details: null,
+      collection_method: 'charge_automatically',
+      created: 0,
+      days_until_due: null,
+      default_payment_method: null,
+      default_source: null,
+      description: null,
+      discount: null,
+      discounts: [],
+      ended_at: null,
+      invoice_settings: {
+        account_tax_ids: null,
+        issuer: {
+          type: 'self',
+        },
+      },
+      latest_invoice: null,
+      livemode: false,
+      next_pending_invoice_item_invoice: null,
+      on_behalf_of: null,
+      pause_collection: null,
+      payment_settings: null,
+      pending_invoice_item_interval: null,
+      pending_setup_intent: null,
+      pending_update: null,
+      schedule: null,
+      start_date: 0,
+      test_clock: null,
+      transfer_data: null,
+      trial_settings: null,
+    },
+  });
+
+  const createMockPriceData = (
+    overrides = {},
+  ): Stripe.PriceCreatedEvent.Data => ({
+    object: {
+      id: 'price_1Q',
+      object: 'price',
+      active: true,
+      billing_scheme: 'per_unit',
+      created: 1733734326,
+      currency: 'usd',
+      custom_unit_amount: null,
+      livemode: false,
+      lookup_key: null,
+      metadata: {},
+      nickname: null,
+      product: 'prod_RLN',
+      recurring: {
+        aggregate_usage: null,
+        interval: 'month',
+        interval_count: 1,
+        meter: null,
+        trial_period_days: null,
+        usage_type: 'licensed',
+      },
+      tax_behavior: 'unspecified',
+      tiers_mode: null,
+      transform_quantity: null,
+      type: 'recurring',
+      unit_amount: 0,
+      unit_amount_decimal: '0',
+      ...overrides,
+    },
+  });
+
+  it('should handle product.updated and price.created webhook events', async () => {
+    const productUpdatedData: Stripe.ProductUpdatedEvent.Data = {
+      object: {
+        id: 'prod_RLN',
+        object: 'product',
+        active: true,
+        created: 1733410584,
+        default_price: null,
+        description: null,
+        images: [],
+        livemode: false,
+        marketing_features: [],
+        metadata: {},
+        name: 'kjnnjkjknkjnjkn',
+        package_dimensions: null,
+        shippable: null,
+        statement_descriptor: null,
+        tax_code: 'txcd_10103001',
+        type: 'service',
+        unit_label: null,
+        updated: 1734694649,
+        url: null,
+      },
+      previous_attributes: {
+        default_price: 'price_1Q',
+        updated: 1733410585,
+      },
+    };
+    const payload = {
+      type: 'product.updated',
+      data: productUpdatedData,
+    };
+
+    await client
+      .post('/billing/webhooks')
+      .set('Authorization', `Bearer ${ACCESS_TOKEN}`)
+      .set('stripe-signature', 'correct-signature')
+      .set('Content-Type', 'application/json')
+      .send(JSON.stringify(payload))
+      .expect(200);
```

Comment:
You're not testing much here when just checking the status code, could you check something else everytime you check a 200, for example modify the response so that it provides the id of the object we just inserted/modified, and then in the test you verify that the request returned a uuid as a response